prisma/base/centralLedger.prisma

```diff
@@ -5,7 +5,7 @@ generator client {
 
 datasource db {
   provider = "mysql"
-  url      = env("CENTRAL_LEDGER_DB_URL")
+  url      = "mysql://test:test@localhost:3306/test"
```

Comment:
Why hardcode?

Is this for local teesting only?

Reply:
These values are just default values and we are overriding them from code. So instead of having an empty string, I just specified the values as test.